src/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+  </head>
+  <body>
+  Hello, world!
+  </body>
```

Comment:
```suggestion
  <body>
    Hello, world!
  </body>
```
Стиль коду важливий. Можна почитати про це тут
https://github.com/mate-academy/style-guides/blob/master/htmlcss.md#indentation

Reply:
Віталій, скажеш, чи правильно все виправив